ports/presentmon/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "presentmon",
-  "version-string": "1.6.0",
+  "version-string": "1.7.0",
```

```suggestion
  "version": "1.7.0",
```
`version `might be better here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version

---

I just kept it as version-string since that was what it was the first time. Any reason to not change @JackBoosY ? I thought I ran into some doc that said version-string was the only supported option right now, but I'm guessing that's not true anymore, and I can't find the stale doc at the moment.

---

`version-string` is the general expression. But we prefer to use more proper filed here. You can see the differences between these filed names. https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-schemes

---

Thanks for explaining. Will fix.